Control coverage/Feature/HybridCompute.md

```diff
@@ -0,0 +1,80 @@
+# HybridCompute
+
+**Resource Type:** Microsoft.HybridCompute/machines
+<!-- TOC depthto:2 depthfrom:2 -->
+
+- [Azure_HybridCompute_DP_Use_Secure_TLS_Version_Trial](#azure_hybridcompute_dp_use_secure_tls_version_trial)
+
+<!-- /TOC -->
+<br/>
+
+___ 
+
+## Azure_HybridCompute_DP_Use_Secure_TLS_Version_Trial
+
+### Display Name 
+[Trial] Use approved version of TLS for ARC Windows Servers
+
+### Rationale 
+TLS provides privacy and data integrity between client and server. Using approved TLS version significantly reduces risks from security design issues and security bugs that may be present in older versions
+
+### Control Settings 
+```json 
+{
+ "ApplicableOsTypes": [
+          "Windows"
+  ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> Azure Policy Configure secure communication protocols (TLS 1.1 or TLS 1.2) is set compliant state to “Compliant”.
```

Comment:
Can we change the grammar for this line. Maybe add quotes for "Configure secure communication protocols".
The way the current line is, it's not making much sense when I'm trying to read it.